apps/mobile/src/urql-shared/client.js

```diff
@@ -20,10 +22,60 @@ const cache = cacheExchange({
 })
 
 const client = createClient({
+  // fetchSubscriptions: true,
```

Comment:
Keeping this commented out?

Reply:
Ah thanks, yes I think I'll remove it. Worth noting that this will probably be on and all we do to handle the Web URQL client subscriptions connection on Web, but didn't support React Native.